homeassistant/components/limitlessled/light.py

```diff
@@ -62,25 +62,14 @@
 
 WHITE = [0, 0]
 
-SUPPORT_LIMITLESSLED_WHITE = (
-    SUPPORT_BRIGHTNESS | SUPPORT_COLOR_TEMP | SUPPORT_EFFECT | SUPPORT_TRANSITION
-)
-SUPPORT_LIMITLESSLED_DIMMER = SUPPORT_BRIGHTNESS | SUPPORT_TRANSITION
-SUPPORT_LIMITLESSLED_RGB = (
-    SUPPORT_BRIGHTNESS
-    | SUPPORT_EFFECT
-    | SUPPORT_FLASH
-    | SUPPORT_COLOR
-    | SUPPORT_TRANSITION
-)
-SUPPORT_LIMITLESSLED_RGBWW = (
-    SUPPORT_BRIGHTNESS
-    | SUPPORT_COLOR_TEMP
-    | SUPPORT_EFFECT
-    | SUPPORT_FLASH
-    | SUPPORT_COLOR
-    | SUPPORT_TRANSITION
-)
+COLOR_MODES_LIMITLESS_WHITE = {COLOR_MODE_COLOR_TEMP}
+SUPPORT_LIMITLESSLED_WHITE = SUPPORT_EFFECT | SUPPORT_TRANSITION
+COLOR_MODES_LIMITLESS_DIMMER = {COLOR_MODE_BRIGHTNESS}
+SUPPORT_LIMITLESSLED_DIMMER = SUPPORT_TRANSITION
+COLOR_MODES_LIMITLESS_RGB = {COLOR_MODE_HS}
+SUPPORT_LIMITLESSLED_RGB = SUPPORT_EFFECT | SUPPORT_FLASH | SUPPORT_TRANSITION
```

Comment:
Several of these will no longer have SUPPORT_BRIGHTNESS. Is that assumed from SUPPORT_TRANSITION, or does this assume that all bulb types have brightness?

Reply:
The `SUPPORT_BRIGHTNESS` flag is deprecated, brightness support is instead implied the color modes: https://developers.home-assistant.io/docs/core/entity/light#color-modes.

Reply:
Note: this branch has conflict following the implementation of `LightEntityFeature`.